common/translations/lang/en.json

```diff
@@ -619,7 +619,7 @@
     "ONBOARD_secure_1_content__1": "Phishers send you a message with a link to a website that looks just like MyCrypto, EtherDelta, Paypal, or your bank, but is not the real website. They steal your information and then steal your money.",
     "ONBOARD_secure_1_content__2": "Install [EAL](https://chrome.google.com/webstore/detail/etheraddresslookup/pdknmigbbbhmllnmgdfalmedcmcefdfn) or [MetaMask](https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn) or [Cryptonite by Metacert](https://chrome.google.com/webstore/detail/cryptonite-by-metacert/keghdcpemohlojlglbiegihkljkgnige) or the [MyCrypto Chrome Extension](https://chrome.google.com/webstore/detail/myetherwallet-cx/nlbmnnijcnlegkjjpcfjclmcfggfefdm) to block malicious websites.",
     "ONBOARD_secure_1_content__3": "Always check the URL: `https://www.mycrypto.com`.",
-    "ONBOARD_secure_1_content__4": "Always make sure the URL bar has `MYCRYPTO, LLC [US]` in green.",
+    "ONBOARD_secure_1_content__4": "Always make sure the URL bar has `MyCrypto, Inc (US)` in green.",
```

Comment:
Good catch, I think we also used MyCrypto LLC elsewhere in the codebase. We'll need to update more places with this.